test/test-window-events.js

```diff
@@ -60,4 +64,20 @@ exports["test browser events"] = function(assert, done) {
   let window = open();
 };
 
+// related to bug 922956
+// https://bugzilla.mozilla.org/show_bug.cgi?id=922956
+exports["test ready event after window.open"] = function (assert, done) {
+  setPref(OPEN_IN_NEW_WINDOW_PREF, 2);
+  setPref(DISABLE_POPUP_PREF, false);
```

Comment:
You need to reset these prefs at the end of the test or it may affect later tests.